From 660e01ce2484c5b0d9e06a4df704a558db528b3a Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 16 Mar 2015 17:48:42 +0000
Subject: sepolicy: Fix gpu access.

---
diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 58312ee..0089f2f 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -1,7 +1,7 @@
 ##########################
 # Devices
 #
-/dev/mali[0-9]*         u:object_r:video_device:s0
+/dev/mali[0-9]*         u:object_r:gpu_device:s0
 
 /dev/bcm2079x           u:object_r:nfc_device:s0
 
--
cgit v0.9.1
