From 89cd3269ce4033c52cde4c320b90ddac6549b65c Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 16 Mar 2015 18:23:28 +0000
Subject: sepolicy: Add selinux domains for daemons

---
diff --git a/BoardConfig.mk b/BoardConfig.mk
index d7f15e7..246c0db 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -121,10 +121,12 @@ BOARD_SEPOLICY_DIRS := \
 
 BOARD_SEPOLICY_UNION := \
 	app.te \
+	cpboot-daemon.te \
 	domain.te \
 	exyrngd.te \
 	file_contexts \
 	file.te \
+	macloader.te \
 	recovery.te \
 	ueventd.te
 
diff --git a/ramdisk/init.universal5430.rc b/ramdisk/init.universal5430.rc
index 4300f83..4104faa 100755
--- a/ramdisk/init.universal5430.rc
+++ b/ramdisk/init.universal5430.rc
@@ -923,6 +923,7 @@ service cpboot-daemon /sbin/cbd -d -txmm72xx -bl -ml -p 13
     class main
     user root
     group radio cache inet misc audio sdcard_rw log sdcard_r shell
+    seclabel u:r:cpboot-daemon:s0
 
 service netmgrd /system/bin/netmgrd
     class late_start
@@ -980,6 +981,7 @@ service exyrngd /system/bin/exyrngd -b
     user root
     group root
     class main
+    seclabel u:r:exyrngd:s0
     oneshot
 
 # virtual sdcard daemon running as media_rw (1023)
@@ -1005,6 +1007,7 @@ service gpsd /system/bin/gpsd -c /system/etc/gps.xml
     user gps
     group system inet net_raw
     ioprio be 0
+    seclabel u:r:gpsd:s0
 
 # on userdebug and eng builds, enable kgdb on the serial console
 on property:ro.debuggable=1
diff --git a/ramdisk/init.universal5430.wifi.rc b/ramdisk/init.universal5430.wifi.rc
index bc7a654..23390f2 100755
--- a/ramdisk/init.universal5430.wifi.rc
+++ b/ramdisk/init.universal5430.wifi.rc
@@ -104,6 +104,7 @@ service hostapd /system/bin/hostapd -dddd /data/hostapd/hostapd.conf
 
 service macloader /system/bin/macloader
     class main
+    seclabel u:r:macloader:s0
     oneshot
 
 on property:init.svc.macloader=stopped
diff --git a/sepolicy/cpboot-daemon.te b/sepolicy/cpboot-daemon.te
new file mode 100644
index 0000000..4ccdb55
--- /dev/null
+++ b/sepolicy/cpboot-daemon.te
@@ -0,0 +1,2 @@
+type cpboot-daemon, domain;
+type cpboot-daemon_exec, exec_type, file_type;
diff --git a/sepolicy/macloader.te b/sepolicy/macloader.te
new file mode 100644
index 0000000..441d1a3
--- /dev/null
+++ b/sepolicy/macloader.te
@@ -0,0 +1,2 @@
+type macloader, domain;
+type macloader_exec, exec_type, file_type;
--
cgit v0.9.1
