From a1fe34ac8740b6dd272c19f6d5e8a1f091d2d4d1 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 16 Mar 2015 16:03:02 +0000
Subject: include: camera: Add missing symbols for Samsung camera.

---
diff --git a/include/camera/CameraParametersExtra.h b/include/camera/CameraParametersExtra.h
index 3dc6a24..572fe4b 100644
--- a/include/camera/CameraParametersExtra.h
+++ b/include/camera/CameraParametersExtra.h
@@ -15,7 +15,21 @@
  */
 
 #define CAMERA_PARAMETERS_EXTRA_C \
-    const char CameraParameters::PIXEL_FORMAT_YUV420SP_NV21[] = "yuv420sp";
+    const char CameraParameters::PIXEL_FORMAT_YUV420SP_NV21[] = "yuv420sp"; \
+    const char CameraParameters::EFFECT_CARTOONIZE[] = "cartoonize"; \
+    const char CameraParameters::EFFECT_POINT_RED_YELLOW[] = "point-red-yellow"; \
+    const char CameraParameters::EFFECT_POINT_GREEN[] = "point-green"; \
+    const char CameraParameters::EFFECT_POINT_BLUE[] = "point-blue"; \
+    const char CameraParameters::EFFECT_VINTAGE_COLD[] = "vintage-cold"; \
+    const char CameraParameters::EFFECT_VINTAGE_WARM[] = "vintage-warm"; \
+    const char CameraParameters::EFFECT_WASHED[] = "washed";
 
 #define CAMERA_PARAMETERS_EXTRA_H \
-    static const char PIXEL_FORMAT_YUV420SP_NV21[]; // NV21
+    static const char PIXEL_FORMAT_YUV420SP_NV21[]; \
+    static const char EFFECT_CARTOONIZE[]; \
+    static const char EFFECT_POINT_RED_YELLOW[]; \
+    static const char EFFECT_POINT_GREEN[]; \
+    static const char EFFECT_POINT_BLUE[]; \
+    static const char EFFECT_VINTAGE_COLD[]; \
+    static const char EFFECT_VINTAGE_WARM[]; \
+    static const char EFFECT_WASHED[];
--
cgit v0.9.1
