From a4950cbc522ec993ad4ae88deb228608a7e5edca Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 16 Mar 2015 16:24:39 +0000
Subject: WIP Fix audio_effects.conf

---
diff --git a/configs/audio/audio_effects.conf b/configs/audio/audio_effects.conf
index 262cb75..dc6a1c7 100644
--- a/configs/audio/audio_effects.conf
+++ b/configs/audio/audio_effects.conf
@@ -164,48 +164,48 @@ effects {
 #                }
 #            }
 
-pre_processing {
-    mic {
-        agc {}
-        ns {}
-        aec{}
-    }
-    camcorder {
+#pre_processing {
+#    mic {
+#        agc {}
 #        ns {}
-    }
-    voice_communication {
-        agc {}
-        ns {}
-        aec {}
-    }
-    sec_voice_recognition{    # for BargeIn
-        agc {}
-        ns {}
-        aec {}
-    }
-    sec_bargein_driving{    # for BargeIn driving
-        agc {}
-        ns {}
-        aec {}
-    }
-    sec_voice_communication{
-        agc {}
-        ns {}
-        aec {}
-    }
-}
-
-
-output_processing {
-
-    voice_call {
-        fens {}
-        lmfv {}
-    }
-
-    sec_voice_communication {
-        fens {}
-        lmfv {}
-    }
-
-}
+#        aec{}
+#    }
+#    camcorder {
+##        ns {}
+#    }
+#    voice_communication {
+#        agc {}
+#        ns {}
+#        aec {}
+#    }
+#    sec_voice_recognition{    # for BargeIn
+#        agc {}
+#        ns {}
+#        aec {}
+#    }
+#    sec_bargein_driving{    # for BargeIn driving
+#        agc {}
+#        ns {}
+#        aec {}
+#    }
+#    sec_voice_communication{
+#        agc {}
+#        ns {}
+#        aec {}
+#    }
+#}
+#
+#
+#output_processing {
+#
+#    voice_call {
+#        fens {}
+#        lmfv {}
+#    }
+#
+#    sec_voice_communication {
+#        fens {}
+#        lmfv {}
+#    }
+#
+#}
--
cgit v0.9.1
