From b188bc56ee1f644cc09b62d9b227df5af2ba0a56 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Fri, 13 Mar 2015 15:28:43 +0000
Subject: recovery: Fix usb access for adb.

---
diff --git a/selinux/prebuilt_file_contexts b/selinux/prebuilt_file_contexts
index b5787de..dc219e9 100644
--- a/selinux/prebuilt_file_contexts
+++ b/selinux/prebuilt_file_contexts
@@ -234,7 +234,7 @@
 /dev/block/platform/dw_mmc.*    u:object_r:dw_mmc_device:s0
 /dev/block/platform/msm_sdcc.1/by-name/param		u:object_r:emmcblk_device:s0
 /dev/block/vnswap0		u:object_r:swap_device:s0
-/dev/bus/usb(/.*)?      u:object_r:usb_bus:s0
+/dev/bus/usb(/.*)?      u:object_r:usb_device:s0
 /dev/diag	u:object_r:diag_device:s0
 /dev/dumpstate	u:object_r:dumpstate_device:s0
 /dev/i2c.*		u:object_r:i2c_device:s0
--
cgit v0.9.1
