From b22a3c3a26d9a3c2aabcdd37b5741cd25bbfacd6 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Thu, 19 Mar 2015 15:28:08 +0000
Subject: BoardConfig: Turn on QC's libm optimizations

---
diff --git a/BoardConfig.mk b/BoardConfig.mk
index 246c0db..80ac70b 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -48,6 +48,9 @@ BOARD_HAS_NO_SELECT_BUTTON := true
 BOARD_HAS_LARGE_FILESYSTEM := true
 TARGET_USERIMAGES_USE_EXT4 := true
 
+### LIBC
+TARGET_USE_QCOM_BIONIC_OPTIMIZATION := true
+
 ### INCLUDE OVERRIDES
 TARGET_SPECIFIC_HEADER_PATH := $(LOCAL_PATH)/include
 
--
cgit v0.9.1
