From c0f05b09852f7ac850ede4b4bc9d15a12ef95f97 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Fri, 13 Mar 2015 15:00:10 +0000
Subject: reocvery: Add sepolicy domain for exyrngd

---
diff --git a/BoardConfig.mk b/BoardConfig.mk
index 258e2a2..d7f15e7 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -122,6 +122,7 @@ BOARD_SEPOLICY_DIRS := \
 BOARD_SEPOLICY_UNION := \
 	app.te \
 	domain.te \
+	exyrngd.te \
 	file_contexts \
 	file.te \
 	recovery.te \
diff --git a/recovery/init.recovery.universal5430.rc b/recovery/init.recovery.universal5430.rc
index 5374553..45b3d78 100644
--- a/recovery/init.recovery.universal5430.rc
+++ b/recovery/init.recovery.universal5430.rc
@@ -14,4 +14,7 @@ on fs
     write /sys/class/android_usb/android0/idVendor 04e8
     write /sys/class/android_usb/android0/idProduct 6860
 
+    mkdir /data/property 0700 root root
+
 service exyrngd /sbin/exyrngd -b
+    seclabel u:r:exyrngd:s0
diff --git a/selinux/prebuilt_file_contexts b/selinux/prebuilt_file_contexts
index 3de9a6e..b5787de 100644
--- a/selinux/prebuilt_file_contexts
+++ b/selinux/prebuilt_file_contexts
@@ -462,6 +462,9 @@
 /efs/gyro_cal_data	u:object_r:sensor_efs_file:s0
 /efs/nv_data.bin(/.*)?	u:object_r:bin_nv_data_efs_file:s0
 
+# Exynrgd
+/sbin/exyrngd u:object_r:exyrngd_exec:s0
+
 #line 1 "vendor/samsung/common/sepolicy/vmware/file_contexts"
 # VMware daemon
 /system/bin/mvpd	u:object_r:vmwared_exec:s0
diff --git a/sepolicy/exyrngd.te b/sepolicy/exyrngd.te
new file mode 100644
index 0000000..fb8961f
--- /dev/null
+++ b/sepolicy/exyrngd.te
@@ -0,0 +1,6 @@
+type exyrngd, domain;
+type exyrngd_exec, exec_type, file_type;
+
+tmpfs_domain(exyrngd)
+
+allow exyrngd tmpfs:chr_file rw_file_perms;
--
cgit v0.9.1
